addons/im_livechat/report/im_livechat_report.py

```diff
@@ -1,49 +1,65 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, fields, models, tools
+from odoo import fields, models, tools
 from odoo.tools import SQL
 
 
-class Im_LivechatReportChannel(models.Model):
-    """ Livechat Support Report on the Channels """
+class Im_LivechatReport(models.Model):
+    """ Livechat Support Report """
 
-    _name = 'im_livechat.report.channel'
-    _description = "Livechat Support Channel Report"
-    _order = 'start_date, technical_name'
+    _name = "im_livechat.report"
+    _description = "Livechat Support Report"
+    _order = "start_date, technical_name"
     _auto = False
 
-    uuid = fields.Char('UUID', readonly=True)
-    channel_id = fields.Many2one('discuss.channel', 'Conversation', readonly=True)
-    channel_name = fields.Char('Channel Name', readonly=True)
-    technical_name = fields.Char('Code', readonly=True)
-    livechat_channel_id = fields.Many2one('im_livechat.channel', 'Channel', readonly=True)
-    start_date = fields.Datetime('Start Date of session', readonly=True)
-    start_hour = fields.Char('Start Hour of session', readonly=True)
-    day_number = fields.Char('Day Number', readonly=True, help="1 is Monday, 7 is Sunday")
-    time_to_answer = fields.Float('Time to answer (sec)', digits=(16, 2), readonly=True, aggregator="avg", help="Average time in seconds to give the first answer to the visitor")
-    start_date_hour = fields.Char('Hour of start Date of session', readonly=True)
-    duration = fields.Float('Average duration', digits=(16, 2), readonly=True, aggregator="avg", help="Duration of the conversation (in seconds)")
-    nbr_speaker = fields.Integer('# of speakers', readonly=True, aggregator="avg", help="Number of different speakers")
-    nbr_message = fields.Integer('Average message', readonly=True, aggregator="avg", help="Number of message in the conversation")
-    is_without_answer = fields.Integer('Session(s) without answer', readonly=True, aggregator="sum",
-                                       help="""A session is without answer if the operator did not answer. 
-                                       If the visitor is also the operator, the session will always be answered.""")
-    days_of_activity = fields.Integer('Days of activity', aggregator="max", readonly=True, help="Number of days since the first session of the operator")
-    is_anonymous = fields.Integer('Is visitor anonymous', readonly=True)
-    country_id = fields.Many2one('res.country', 'Country of the visitor', readonly=True)
-    is_happy = fields.Integer('Visitor is Happy', readonly=True)
-    rating = fields.Integer('Rating', aggregator="avg", readonly=True)
+    uuid = fields.Char("UUID", readonly=True)
+    channel_id = fields.Many2one("discuss.channel", "Conversation", readonly=True)
```

Comment:
The file has mostly useless formatting diff on unchanged lines, it makes the PR quite bigger and harder to review than it should.